projects/plugins/boost/app/assets/src/js/features/critical-css/status/status.module.scss

```diff
@@ -68,4 +68,12 @@
 			fill: $jetpack-green;
 		}
 	}
+
+	.regenerate-button:global(.components-button) {
+		&:disabled {
+			background: transparent;
+			opacity: .5;
+			cursor: not-allowed;
```

Comment:
Though a nice touch, I think changing the cursor is a bit inconsistent. We do not change cursors in Boost or Jetpack and this will fall a bit out of the ordinary.

Reply:
You're right, we don't have anything like that throughout the UI. I've removed it.